templates/components/faceted-search/facets/range.html

```diff
@@ -23,11 +23,11 @@ <h5 class="accordion-title">
     <div id="facetedSearch-content--{{hyphenate facet }}" class="accordion-content {{#unless ../start_collapsed }} is-open {{/unless}}">
         <form id="facet-range-form" class="form" method="get" data-faceted-search-range novalidate>
             {{#each current_selected_items}}
-                <input type="hidden" name="{{ param_name }}[]" value="{{ param_value }}"/>
```

Comment:
This could actually be problematic. What `param_name` values actually come out here? I can verify that none are treated as an array, but if any are actually supposed to be arrays, this change will break that.

Reply:
I'll undo that change because other things are using []. But for the record,

*Without* touching range I can get a url parameter string like:
`?_bc_fsnf=1&brand[]=36&brand[]=35&Color[]=Black&Color[]=Blue`

Which has identical results to:
`?_bc_fsnf=1&brand=36&brand=35&Color=Black&Color=Blue`

Values aren't being added to the parameters in any sort of array-like structure. Their keys are simply repeated.

Specific to changes in `range.html`, with the brackets `?brand=37&brand=36&search_query=&min_price=1&max_price=99` returns the same results as pre-change `?brand[]=37&brand[]=36&search_query=&min_price=1&max_price=99`